examples/elk/README.md

```diff
@@ -0,0 +1,46 @@
+# converge-elk
+
+A Converge example that sets up a single node docker-based [ELK](https://www.elastic.co/webinars/introduction-elk-stack) stack.
+
+[Filebeat](https://www.elastic.co/products/beats/filebeat) is used instead of [Logstash](https://www.elastic.co/products/logstash) for the log collection component.
+
+## Usage
+
+### Vagrant
+
+In the [Vagrantfile](./Vagrantfile), change the file provisioner source to point to a version of the converge binary built with the `linux/amd64` OS architecture.
```

Comment:
we'll want to update this again when https://github.com/ChrisAubuchon/vagrant-converge is ready but I went ahead and scripted out the converge download as part of vagrant provisioning. now you only have to run `vagrant up`.